lib/convert.js

```diff
@@ -28,13 +28,14 @@
         Q = require("q");
 
     // Constants
-    var DEFAULT_IMAGE_QUALITY           = 90,
+    var DEFAULT_IMAGE_QUALITY           = "90",
         BACKGROUND_COLOR_FOR_FLATTENING = "#fff";
 
     function _getConvertArgumentsForSettings(settings, binaryPaths) {
         var args    = [],
             format  = settings.format,
-            quality = settings.quality ? String(settings.quality) : null,
+            intQuality = Number.parseInt(settings.quality),
```

Comment:
My understanding is the quality parameter to FLITE needs to be in an integer, and flooring seems OK.  I don't think in practice we ever accept float values